packages/atlas/src/hooks/useSegmentAnalytics.ts

```diff
@@ -35,13 +35,15 @@ export const useSegmentAnalytics = () => {
 
   const identifyUser = useCallback(
     (email = 'no data') => {
+      if (!analytics) return
```

Comment:
It will never be null if we used the correct hook

Reply:
> It will never be null if we used the correct hook

but if I don't wont to have segment at all and I don't have a key this will stay as null:
`() => ({ analytics: writeKey ? AnalyticsBrowser.load({ writeKey }) : null }),`